python-sdk/nuscenes/eval/predict/README.md

```diff
@@ -15,33 +15,33 @@ A trajectory is a sequence of x-y locations. For this challenge, the predictions
 Note that the evaluation server can still be used to benchmark your results after the challenge period.
 
 ## Submission rules
 ### Prediction-specific rules
 * The user can submit up to 25 proposed future trajectories, called `modes`, for each agent along with a probability the agent follows that proposal. Our metrics (explained below) will measure how well this proposed set of trajectories matches the ground truth.
 * Unlike previous challenges, the user will not submit their submissions to the eval server. Instead, we will run the user's model
-on the test set for them. To make this possible, the user should modify the `do_inference_for_submission` function in the [inference script](https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes-predict-challenge/python-sdk/nuscenes/eval/predict/do_inference.py)
+on the test set for them. This is because we do not want to interfere with the [nuScenes detection challenge](https://www.nuscenes.org/object-detection?externalData=all&mapData=all&modalities=Any) by exposing the test set annotations. In order for us to run their model, the user should modify the `do_inference_for_submission` function in the [inference script](https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes-predict-challenge/python-sdk/nuscenes/eval/predict/do_inference.py)
 so that their model is used.
 * Up to two seconds of past history can be used to predict the future trajectory for each agent.
 * Every submission to the challenge must be accompanied by a technical report describing the method in sufficient detail to allow for independent verification.
 
 ### General rules
 * We release annotations for the train and val set, but not for the test set.
 * We release sensor data for train, val and test set.
-* Top leaderboard entries and their papers will be manually reviewed.
+* Top leaderboard entries and their papers will be manually reviewed to ensure no cheating was done.
 * Each user or team can have at most one one account on the evaluation server.
-* Each user or team can submit at most 3 results. These results must come from different models, rather than submitting results from the same model at different training epochs or with slightly different parameters.
-* Any attempt to circumvent these rules will result in a permanent ban of the team or company from all nuScenes challenges.
+* Each user or team can submit at most 3 results. These results must come from different models, rather than submitting results from the same model at different training epochs or with slightly weights or hyperparameter values.
+* Any attempt to make more submissions than allowed will result in a permanent ban of the team or company from all nuScenes challenges.
 
 ## Results format
 The `do_inference_for_submission` function must produce a list of [`Predictions`](https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes-predict-challenge/python-sdk/nuscenes/eval/predict/data_classes.py). A `Prediction` has the following components:
```

Comment:
Below you write "l2 distance". I suggest capitalizing it to "L2".

Reply:
Thanks Holger!